docs/getting-started/server/admin.md

```diff
@@ -1,19 +1,51 @@
-# Admin Portal
+# Bitwarden Portal
 
-## Configuring users
+## Configuring access
 
-Admin Portal authentication is done entirely through a passwordless flow, using a link sent through
-email. The email address must be listed in the `adminSettings:admins` user secret to be authorized.
+### Authentication
+
+Bitwarden Portal authentication is done entirely through a passwordless flow, using a link sent
+through email. The email address must be listed in the `adminSettings:admins` user secret to be
+authorized.
 
 If you’ve followed the [Server Setup Guide](./guide.md) this should already be configured and will
 default to `admin@localhost`. If not, please go back and configure it now.
```

Comment:
Do we want to update https://github.com/bitwarden/server/blob/master/dev/secrets.json.example as well?